Jenkinsfile

```
@@ -112,6 +112,13 @@ for (i = 0; i < buildTypes.size(); i++) {
               qualityGates: [
                 [threshold: 1, type: 'TOTAL', unstable: true],
               ]])
+             recordIssues([tool: esLint(pattern: '**/target/eslint-warnings.xml'),
+              sourceCodeEncoding: 'UTF-8',
+              skipBlames: true,
+              trendChartType: 'TOOLS_ONLY',
+              qualityGates: [
+                [threshold: 1, type: 'NEW', unstable: true],
```

new is almost never what you want, it's not reproduce-able outside the CI system
```suggestion
                [threshold: 1, type: 'TOTAL', unstable: true],
```